hazelcast/src/main/java/com/hazelcast/internal/diagnostics/HealthMonitor.java

```diff
@@ -460,9 +460,10 @@ private void renderNativeMemory() {
                 sb.append("native.meta.memory.used=")
                         .append(numberToUnit(usedMeta)).append(", ");
                 sb.append("native.meta.memory.free=")
-                        .append(numberToUnit(maxMeta - usedMeta)).append(", ");
+                        .append(usedMeta >= maxMeta ? 0 : numberToUnit(maxMeta - usedMeta)).append(", ");
                 sb.append("native.meta.memory.percentage=")
-                        .append(percentageString(PERCENTAGE_MULTIPLIER * usedMeta / (usedNative + usedMeta))).append(", ");
+                        .append(usedMeta >= maxMeta ? percentageString(PERCENTAGE_MULTIPLIER)
```

Comment:
What is the reason to show `native.meta.memory.percentage` as `100%` when `usedMeta >= maxMeta`? The original metric shows the percentage of used native memory that is allocated to metadata. So if `usedMeta` exceeds maximum configured for metadata, then it will probably display a percentage which is above the configured one, but it is definitely not 100% of all used native memory. wdyt?

Reply:
right, considered it also but there is no known upper limit to calculate it.

Reply:
BTW it means, 100% of metadata memory quota has beed used, not all native memory.